aws/redshift/queries/cs_fundamentals_started.sql

```diff
@@ -0,0 +1,202 @@
+-- New CSF teachers by month
+SELECT COUNT(DISTINCT user_id_teacher) value,
+       'New CSF teachers' metric
+FROM (SELECT user_id_teacher,
+             user_id_student,
+             first_script_started_at,
+             ROW_NUMBER() OVER (PARTITION BY user_id_teacher ORDER BY first_script_started_at ASC) student_progress_rank
+      FROM (SELECT se.user_id user_id_teacher,
+                   us.user_id user_id_student,
+                   MIN(us.started_at) first_script_started_at
+            FROM dashboard_production.sections se
+              JOIN dashboard_production.followers f ON f.section_id = se.id
+              JOIN dashboard_production.user_scripts us ON us.user_id = f.student_user_id
+            WHERE se.script_id IN (1,17,18,19,23,236,237,238,239,240,241,258,259)
+            AND   us.script_id IN (1,17,18,19,23,236,237,238,239,240,241,258,259)
+            AND   us.started_at IS NOT NULL
+            GROUP BY 1,
+                     2))
+WHERE student_progress_rank = 5
+AND   DATE_PART(month,first_script_started_at) = CASE WHEN DATE_PART(month,getdate()) = 1 THEN 12 ELSE DATE_PART(month,getdate()) - 1 END
+AND   DATE_PART(year,first_script_started_at) = CASE WHEN DATE_PART(month,getdate()) = 1 THEN DATE_PART(year,getdate()) - 1 ELSE DATE_PART(year,getdate()) END
+GROUP BY 2
+
+UNION ALL
+
+-- New CSF students by month
+SELECT COUNT(DISTINCT user_id) value,
+       'New CSF students' metric
+FROM (SELECT user_id,
+             MIN(started_at) date_first_activity
+      FROM (SELECT us.user_id,
+                   us.started_at
+            FROM dashboard_production.user_scripts us
+              JOIN dashboard_production.users u ON u.id = us.user_id
+            WHERE us.script_id IN (1,17,18,19,23,236,237,238,239,240,241,258,259)
+            AND   u.user_type = 'student')
+      GROUP BY 1)
+WHERE DATE_PART(month,date_first_activity) = CASE WHEN DATE_PART(month,getdate()) = 1 THEN 12 ELSE DATE_PART(month,getdate()) - 1 END
+AND   DATE_PART(year,date_first_activity) = CASE WHEN DATE_PART(month,getdate()) = 1 THEN DATE_PART(year,getdate()) - 1 ELSE DATE_PART(year,getdate()) END
+GROUP BY 2
+
+UNION ALL
+
+-- New CSF students by month, percent female
+SELECT COUNT(DISTINCT CASE WHEN gender = 'f' THEN user_id END)::FLOAT/ COUNT(DISTINCT user_id) value,
+       'New CSF students, % female' metric
+FROM (SELECT user_id,
+             gender,
+             MIN(started_at) date_first_activity
+      FROM (SELECT us.user_id,
+                   us.started_at,
+                   u.gender
+            FROM dashboard_production.user_scripts us
+              JOIN dashboard_production_pii.users u ON u.id = us.user_id
+            WHERE us.script_id IN (1,17,18,19,23,236,237,238,239,240,241,258,259)
```

Comment:
Might be worth annotating what these numbers refer to (or derive them from a subquery that makes more sense if possible).